### ISSUE TITLE
Shaman weapon enchant bug

### ISSUE THREAD

**Issue by Dainina** [kvakvs/BuffomatClassic#39]
Hey and lovely addon!
Been using it on my Shaman and at some point it stopped closing after buffing Wep enchants.

So I'll press the button to re-apply Flametongue or Rockbiter but it doesn't "clear". I hope that makes sense.

Thank you!

**Comment by kvakvs** [comment]
If you open the addon by `/bom` or keyboard shortcut it will remain open.

Easiest solution is to go to Keybinds, and assign a key. If it shows when you don't want it, press that key to show or hide. It will remember and will hide again like the usual.

**Comment by WorkForPizza** [comment]
I'm getting this behavior too on classic era - buffomat will suggest a rebuff of Rockbiter immediately after I cast it and never seems to "clear". I tried tracking it down tonight, and I think it comes down to playerUnit.knownBuffs in taskScanModule:GetActiveAuraAndSeal not being populated with weapon enchants, because the UnitAura api call doesn't return info about weapon enchants. I couldn't find a different way of getting that info, besides going through the GetWeaponEnchantInfo api which gives an enchantId instead of a spellId. Or maybe I'm way off base and looking in the wrong part of the code. Any ideas on this @kvakvs?

**Comment by kvakvs** [comment]
In a raid buffomat will delay the scan, so buff it and wait for a second or 3. Solo this should not be happening.

**Comment by WorkForPizza** [comment]
Right, this was on a brand new shaman in the leveling zone. I put in debug prints and saw many scans happen and each time the weapon enchant was added to the table even though it was already applied to my weapon and still at the max duration of 5 min.

**Comment by kvakvs** [comment]
There are a group of settings in the Config (named Time60, Time300, Time600, Time1800 and Time3600), you can find them in the options window or in the savefile whatever is more comfortable for you. If you accidentally set Time300 for 5min buffs too high, it will ask to rebuff very soon.

**Comment by WorkForPizza** [comment]
Good thought, but it looks like I have those set at their minimum values already. 
![Screenshot 2023-04-28 at 9 19 17 AM](https://user-images.githubusercontent.com/10823915/235173280-5e7db7ed-6e6e-47c6-9bb8-81257b9077df.png)
